src/App.tsx

```diff
@@ -1,14 +1,55 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
+
+import debounce from 'lodash.debounce';
+
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { PeopleList } from './components/PeopleList/PeopleList';
+
+const filterPeople = (query: string) => {
+  const normalizedQuery = query.trim().toLowerCase();
+
+  return peopleFromServer.filter(person => {
+    const normalizedName = person.name.trim().toLowerCase();
+
+    return normalizedName.includes(normalizedQuery);
+  });
+};
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [delay] = useState(1000);
```

Comment:
Not need to create state with delay variable. Move the variable outside the component instead